### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by fayejf**
# What does this PR do ?

update readme regarding container version and python version

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Review by titu1994 — DISMISSED** [review]
This looks good !

**Review by XuesongYang — COMMENTED** [review]
Thanks for the update! Besides these changes, I guess we have to update ones to display correct info in PyPi repository. For example, we may need to change below,
1. https://github.com/NVIDIA/NeMo/blob/main/setup.py#L216-L219;
2. https://github.com/NVIDIA/NeMo/blob/main/pyproject.toml#L19

**Comment by fayejf** [comment]
> Thanks for the update! Besides these changes, I guess we have to update ones to display correct info in PyPi repository. For example, we may need to change below,
> 
> 1. https://github.com/NVIDIA/NeMo/blob/main/setup.py#L216-L219;
> 2. https://github.com/NVIDIA/NeMo/blob/main/pyproject.toml#L19

Make sense. Thanks for pointing out. Updating.

**Comment by titu1994** [comment]
Probably best to update it to 3.10+ everywhere so that it gets tested more

File: setup.py [hunk 1]
```diff
@@ -217,6 +217,7 @@ def finalize_options(self):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
```
Comment (titu1994) [inline setup.py:220]: Let's remove 3.8 from the above ?

**Review by titu1994 — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (fayejf) [reply setup.py:219]: updated.

**Comment by fayejf** [comment]
> Probably best to update it to 3.10+ everywhere so that it gets tested more

 Please approve again. The reason I hesitate to update to 3.10+ everywhere is I doubt phuoc was testing with 3.10. It's better to ask him to QA test on 3.10 in next release in case anything breaks.

File: setup.py [hunk 2]
```diff
@@ -215,8 +215,8 @@ def finalize_options(self):
         'License :: OSI Approved :: Apache Software License',
         # Supported python versions
         'Programming Language :: Python :: 3',
```
Comment (XuesongYang) [inline setup.py:217]: do we want to keep this?

**Review by XuesongYang — COMMENTED** [review]
Thanks! LGTM. rotating to @titu1994 and @ericharper for a final walk through.

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**